README.md

```diff
@@ -64,6 +64,7 @@ Create a [personal access token](https://docs.gitlab.com/ce/user/profile/persona
 | `gitlabApiPathPrefix` | The GitLab API prefix.                                                                                                                         | `GL_PREFIX` or `GITLAB_PREFIX` environment variable or CI provided environment variables if running on [GitLab CI/CD](https://docs.gitlab.com/ee/ci) or `/api/v4`.      |
 | `assets`              | An array of files to upload to the release. See [assets](#assets).                                                                             | -                                                                                                                                                                       |
 | `milestones`          | An array of milestone titles to associate to the release. See [GitLab Release API](https://docs.gitlab.com/ee/api/releases/#create-a-release). | -                                                                                                                                                                       |
+| `postComments`          | Post comments to issues and MRs associated with a release. | `false`                                                                                                                                                                       |
```

Comment:
I went for the fully backwards compatible way here, but we might also default to `true`?

Reply:
I hesitate slightly here because this differs from the [options available in the github plugin](https://github.com/semantic-release/github/#options). i would ideally like the functionality between the two plugins to be as similar as possible. 

that plugin currently allows changing the templated messages for `successComment` and `failComment`, allowing the comments to be disabled by setting the options to `false`. while i don't necessarily like that the templated approach would be a more complicated implementation, i do like the alignment between the two plugins with that approach.

Reply:
i like that defaulting to `false` with this implementation could make this available without it being a breaking change. if we switched from `postComments` to an approach that exposed the `successComment` and `failComment`, i think defaulting to `false` would be more difficult to justify. however, enabling the comments by default would be unexpected to existing users and likely not desired by at least some consumers of the plugin. 

i think i would lean toward enabling the comments by default if we do decide to go with that approach, but making it a breaking change.

Reply:
All done, happy to get some feedback @travi!